src/indexes.rs

```diff
@@ -480,7 +480,7 @@ impl Index {
         .await
     }
 
-    /// Add a list of [Document]s or replace them if they already exist.
+    /// Add a list of \[Document\]s or replace them if they already exist.
```

Comment:
```suggestion
    /// Add a list of documents or replace them if they already exist.
```

Reply:
I wonder if I should commit the suggestions separately or change the index.rs locally then commit in one commit.

Reply:
One commit is better!